lib/cell.rb

```diff
@@ -10,4 +10,9 @@ def initialize(coordinate)
   def empty?
     @empty
   end
+
+  def place_ship(ship)
+    @ship = ship
```

Comment:
This one had us tripped up for a min. We thought we had tried this previously. Good reminder that it doesn't hurt to trust your instincts and try what you feel is correct a second or third time. Who knows, there could have been a typo the first go around.